zboxcore/sdk/allocation.go

```diff
@@ -2067,7 +2067,7 @@ func (a *Allocation) getConsensuses() (fullConsensus, consensusThreshold int) {
 		return a.DataShards, a.DataShards
 	}
 
-	return a.DataShards + a.ParityShards, a.DataShards + 1
+	return a.DataShards + a.ParityShards, a.DataShards
```

Comment:
What's the reason of this change? The `+ 1` is there for a reason, in Erasure encoding, minimum of data+1 number of shards are needed to correctly recover the data, so it is the consensusThreshold. You should not be removing this `+ 1`.